### DISCUSSION TITLE
Tesla Style Solar Power Card

### DISCUSSION THREAD

**Discussion by JohanAlvedal** [wlcrs/huawei_solar#622] [General]
Is solar_to_house really correct or am I misunderstanding?
https://github.com/wlcrs/huawei_solar/wiki/Visualisations

My solar cells give 95W but solar_to_house shows 3kw

![tsc](https://github.com/wlcrs/huawei_solar/assets/25838439/aba0216f-219f-49cc-b2e5-60201036a790)

**Comment by Roving-Ronin** [comment]
@JohanAlvedal 

Personally I'd suggest ditching that card and looking at:

Power Flow Card Plus (i.e. shows live Power / Watts)
https://github.com/flixlix/power-flow-card-plus

and for Energy (kWh used that day.. if you have meters setup for that) see:

Energy Flow Card Plus
https://github.com/flixlix/energy-flow-card-plus

If you don't have meters setup with cycle:daily (to roll over daily...or hourly/weekly/monthly/yearly etc), I'd also suggest looking at:

Energy Meter - that you can use instead of utility_meter and allows easily adding costs tracking and tariffs etc.
https://github.com/zeronounours/HA-custom-component-energy-meter
